myapp/package.json

```diff
@@ -22,6 +22,7 @@
     "bcrypt": "^5.1.1",
     "body-parser": "^1.20.2",
     "cors": "^2.8.5",
+    "dotenv": "^16.4.5",
```

Comment:
dotenv package is not necessary. The plan is to 
- setup ENV variable outside of Nodejs, and is done in docker stage
- refer to [compose.sh](https://github.com/cwwojin/FORK_backend/blob/main/compose.sh), [docker-compose.yml](https://github.com/cwwojin/FORK_backend/blob/main/docker-compose.yml), [Dockerfile](https://github.com/cwwojin/FORK_backend/blob/main/Dockerfile)